.github/workflows/latest-git-tag.yml

```diff
@@ -0,0 +1,19 @@
+# Create or update a latest git tag when releasing a stable vesrsin of Meilisearch
+name: Update latest git tag
+on:
+  workflow_dispatch:
+  release:
+    types: [released]
+
+jobs:
+  update-latest-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: rickstaa/action-create-tag@v1
+        with:
+          tag: "latest"
+          message: "Latest stable release of Meilisearch"
+          # Move the tag if `latest` already exists
+          force_push_tag: true
```

Comment:
Does this remove the previous latest tag?
That's why we don't need to delete it?

Reply:
It moves the tag, you cannot push two similar tags in the same git repo. So it uses the `--force` command
https://stackoverflow.com/questions/8044583/how-can-i-move-a-tag-on-a-git-branch-to-a-different-commit

Reply:
Nice thank you!